cve_bin_tool/OutputEngine.py

```diff
@@ -96,78 +93,45 @@ def output_csv(self, outfile):
         writer.writeheader()
         writer.writerows(self.formatted_output)
 
-    def write_console(self, productname, version, cve_number, cve_severity, outfile):
-        """ Output Module, Version, CVE_Number, Severity to the console in tabular form"""
-
-        if len(productname) > (self.PRODUCTNAME_MAX - len(self.DOTS)):
-            productname = (
-                productname[: self.PRODUCTNAME_MAX - len(self.DOTS)] + self.DOTS
+    def output_console(self, console=Console()):
+        """ Output list of CVEs in a tabular format with color support """
+
+        # table instance
+        table = Table()
+
+        # Add Head Columns to the Table
+        table.add_column("Product")
+        table.add_column("Version")
+        table.add_column("CVE Number")
+        table.add_column("Severity")
+
+        # colors provide color name according to the severity
+        colors = {"CRITICAL": "red", "HIGH": "blue", "MEDIUM": "yellow", "LOW": "green"}
```

Comment:
Failing on windows only during the test console output.  I'm guessing one of the colours isn't available in the limited windows palette?

Reply:
@terriko rich does not fail on unsupported colours it just skips that. The error was due to the encoding. I found out that open by default opens with `utf-8` encoding but in case of windows, it is 8-bit set characters. So we need to force the encoding. Moreover rich changes some chars based on the operating system and terminal. So I had to remove those chars and now I'm testing only the presence of data with some fixed chars which don't change often.